src/seedu/addressbook/commands/Command.java

```diff
@@ -26,6 +26,15 @@ public Command(int targetIndex) {
     protected Command() {
     }
 
+    /**
+     * @return true if the object calling the method is an instance of a mutating Command subclass
```

Comment:
Good job for writing the header comment. BTW, for a method with only return statement, you can instead write it as `Returns true if ...` and remove `@return`.